src/actions/AuthActions.tsx

```diff
@@ -1,12 +1,19 @@
 /**
```

Comment:
Deze comment staat volgens mij niet op de juiste locatie, hij moet boven regel 8 beginnen

Reply:
Implemented in : commit:
edbc9a5